signup/signupController.js

```diff
@@ -167,6 +167,10 @@ define('signupController', [], function() {
 			}
 		};
 
+		$scope.isEmailRequired = () => {
+			return $scope.sharedProperties.isSelectedSiteVirtual == true;
```

Comment:
Could we switch this from `==` to `===`?

Reply:
We could.. I opted for the `==` because I know behind-the-scenes the `isSelectedSiteVirtual` property comes from the `isVirtual` field in the DB, and I didn't want to risk a situation where `isSelectedSiteVirtual` actually contained 0 or 1 instead of false or true... So I thought the truthy equality was "safer" in that sense. But I could look at the code and see if `isSelectedSiteVirtual` takes on `true`/`false`

Reply:
Took a look, the PHP takes of doing a `(boolean)` cast so we should be good to do `===`

Reply:
Cool cool. I was just logging it out, and it seemed to always be true or false. Thanks for double-checking.